app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -1,51 +1,32 @@
-import React, { useState } from "react";
+import React from "react";
 
-import { Modal } from "neetoui";
+import { Modal, Typography, Button, Toastr } from "neetouiv2";
 
-import notesApi from "apis/notes";
-
-export default function DeleteAlert({ refetch, onClose, selectedNoteIds }) {
-  const [deleting, setDeleting] = useState(false);
-  const handleDelete = async () => {
-    try {
-      setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
-      onClose();
-      refetch();
-    } catch (error) {
-      logger.error(error);
-    } finally {
-      setDeleting(false);
-    }
+export default function DeleteAlert({ showToast, setShowToast }) {
+  const handleDelete = () => {
+    Toastr.success("Note successfully deleted.");
+    setShowToast(false);
   };
   return (
-    <Modal
-      isOpen
-      size="small"
-      autoHeight
-      showFooter
-      submitButtonProps={{
-        style: "danger",
-        label: "Continue anyway",
-        loading: deleting,
-        onClick: handleDelete
-      }}
-      onClose={onClose}
-    >
-      <div className="flex">
-        <div className="flex items-center justify-center flex-shrink-0 w-10 h-10 bg-red-100 rounded-full">
-          <i className="text-red-500 ri-alarm-warning-fill ri-lg"></i>
-        </div>
-
-        <div className="ml-4">
-          <h3 className="mb-2 text-lg font-medium text-gray-700">
-            Delete {selectedNoteIds.length} notes?
-          </h3>
-          <div className="text-sm leading-5 text-gray-500">
-            Are you sure you want to continue? This cannot be undone.
-          </div>
-        </div>
-      </div>
+    <Modal isOpen={showToast} onClose={() => setShowToast(false)}>
+      <Modal.Header>
+        <Typography style="h2">Delete Note</Typography>
+      </Modal.Header>
+      <Modal.Body>
+        <Typography style="body2" lineHeight="normal">
+          Are you sure you want to delete the note? This action cannot be
+          undone.
+        </Typography>
+      </Modal.Body>
+      <Modal.Footer className="space-x-2">
+        <Button label="Continue" onClick={handleDelete} size="large" />
+        <Button
+          style="text"
+          label="Cancel"
+          onClick={() => setShowToast(false)}
+          size="large"
+        />
+      </Modal.Footer>
```

Comment:
Please create a single reusable component for both, deleting contacts and notes.